README.md

```diff
@@ -1,4 +1,4 @@
-# jsx-pdf
+[![jsx_pdf.jpg](https://s15.postimg.cc/wcofwdugb/jsx_pdf.jpg)](https://postimg.cc/image/7jevvqbfr/)
```

Comment:
Looks great! Very bold 😁 

A couple of small tweaks:
1. The img should have an appropriate alt text (the stuff inside the square brackets).  How about "A jsx-pdf logo"
2. We should probably upload it to github images (you can drag the image into a github editor to achieve this).